ydb/core/tx/schemeshard/ut_base/ut_base.cpp

```diff
@@ -10477,45 +10477,27 @@ Y_UNIT_TEST_SUITE(TSchemeShardTest) {
                             NLs::NoMaxPartitionsCount
                             });
 
-        // request without token
+        // request direct alter of forbidden fields of indexImplTable
         TestAlterTable(runtime, ++txId, "/MyRoot/table/indexByValue/", R"(
-                        Name: "indexImplTable"
-                        PartitionConfig {
-                            PartitioningPolicy {
-                                MinPartitionsCount: 1
-                                SizeToSplit: 100502
-                                FastSplitSettings {
-                                    SizeThreshold: 100502
-                                    RowCountThreshold: 100502
-                                }
-                            }
-                        }
-                    )", {NKikimrScheme::StatusNameConflict});
-
-        {  // request with not a super user token
-            auto request = AlterTableRequest(++txId, "/MyRoot/table/indexByValue/", R"(
-                        Name: "indexImplTable"
-                        PartitionConfig {
-                            PartitioningPolicy {
-                                MinPartitionsCount: 1
-                                SizeToSplit: 100501
-                                FastSplitSettings {
-                                    SizeThreshold: 100501
-                                    RowCountThreshold: 100501
-                                }
-                            }
+                Name: "indexImplTable"
+                KeyColumnNames: ["key", "value"]
+                PartitionConfig {
+                    PartitioningPolicy {
+                        MinPartitionsCount: 1
+                        SizeToSplit: 100502
+                        FastSplitSettings {
+                            SizeThreshold: 100502
+                            RowCountThreshold: 100502
                         }
-               )");
-
-            auto wellCookedToken = NACLib::TUserToken(TVector<TString>{"not-a-root@builtin"});
-            request->Record.SetUserToken(wellCookedToken.SerializeAsString());
```

Comment:
Why cases with requests with explicit tokens are gone? Check for admin token in alter table is still there.